java/integTest/src/test/java/glide/PubSubTests.java

```diff
@@ -1468,4 +1468,60 @@ public void pubsub_channels_and_numpat_and_numsub_in_transaction(boolean standal
                 },
                 result);
     }
+
+    @SneakyThrows
+    @ParameterizedTest(name = "standalone = {0}")
+    @ValueSource(booleans = {false})
+    public void pubsub_shard_channels(boolean standalone) {
```

Comment:
Don't parametrize - test cluster client only
```suggestion
    @Test
    public void pubsub_shard_channels() {
```